actionpack/lib/system_test_case.rb

```diff
@@ -0,0 +1,57 @@
+require "system_testing/test_helper"
+require "system_testing/driver_adapter"
+
+module Rails
```

Comment:
@dhh `Rails::SystemTestCase` inherits from `ActionDispatch::IntegrationTest` so we don't have to rewrite all the routing and url helpers handling.